database/database.go

```diff
@@ -53,8 +53,9 @@ func cause(err error) error {
 
 // Options are configuration options for the database.
 type Options struct {
-	Database string
-	ValueDir string
+	Database                  string
+	ValueDir                  string
+	BadgerValueLogLoadingMode string
```

Comment:
In badger this is called `FileLoadingMode`, it should probably be `BadgerFileLoadingMode` or just `FileLoadingMode`, this option will be ignored by the rest of the implementations.

Reply:
In Badger the type is called `FileLoadingMode`, but the attribute is called `ValueLogLoadingMode` in the Badger Options. There are multiple attributes that have the type `FileLoadingMode`. 

```
type Options struct {
	Dir      string
	ValueDir string

	SyncWrites          bool
	TableLoadingMode    options.FileLoadingMode
	ValueLogLoadingMode options.FileLoadingMode
	NumVersionsToKeep   int
	ReadOnly            bool
	Truncate            bool
...
```

Reply:
We should use the same mode for both. In badger Table and Value mean something, but in nosql they don't really if we don't take into account that `ValueDir` option that was added back in the day. Unless we want to define independent values for Table and Value we should just call it FileLoadingMode.

Reply:
I changed it to `FileLoadingMode`. But if we need to go back later and define different values for Table and ValueLog I'm gonna blame you.